### PR TITLE
Type1 Workload

### PR DESCRIPTION
**Pull request by VikasKumarC**
Signed-off-by: Vikas Kumar Choudhary <vikas.kumar@seagate.com>

## Problem Statement

-   Problem statement

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

File: workload/s3/s3api/object_operations_type1.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        0K: 200
+        1K: 679
+        10Kb: 1884
+        100Kb: 1787
+        1M: 1820
+        10M: 1670
+        100M: 156
+        1G: 3
+        2G: 1
+    write_percentage: 100  # Percentage of data to fill the storage.
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: Add total samples as discussed.

File: workload/s3/s3api/object_operations_type1.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    read_percentage: 100  # Percentage of data to be used to read from storage.
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: Please remove write/read/delete/cleanup percentage. not required for type 1

File: workload/s3/s3api/object_operations_type1.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    delete_percentage: 100  # Percentage of data to be deleted from storage.
+    cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+    #  If None then it will be fetched from cluster(cortx specific.)
+    total_storage_size: None  # Total available storage from cluster or user to be used.
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: total storage size also not required.

File: workload/s3/s3api/object_operations_type1.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    min_runtime: 30d  # Minimum execution durations to mark workload pass.
+    sessions_per_node: 20  # Number of sessions per node.
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: Session per node can be changed to 2.

File: workload/s3/s3api/object_operations_type1.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    background_delete: False  # Enable/Disable background delete(Cortx specific).
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: background delete also can be added when supported. 
Can be removed for now.

File: workload/s3/s3api/object_operations_type1.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    tool: s3api
+    operation: "type1_object_ops"
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_type1.yaml]: Add new line at end of file.

**Review by DhobaleVishal — APPROVED** [review]
LGTM, I can see all comments resolved.

File: workload/s3/s3api/object_operations_type1.yaml (hunk 4 above)
Comment (VikasKumarC) [reply workload/s3/s3api/object_operations_type1.yaml]: This is customizable parameter, user can update same? 
May I know why it should be default 2?

**Review by ketanarlulkar — APPROVED** [review]

**Review by priyankaborawake — APPROVED** [review]

File: workload/s3/s3api/object_operations_type1.yaml (hunk 4 above)
Comment (priyankaborawake) [reply workload/s3/s3api/object_operations_type1.yaml]: Please keep it configurable through jenkins. 
We are currently executing the test with 2 sessions per worker node

**Pull request merged**